coordinator/impl/shard_controller.go

```diff
@@ -497,9 +501,9 @@ func (s *shardController) internalNewTermAndAddFollower(ctx context.Context, nod
 func (s *shardController) newTermQuorum() (map[model.ServerAddress]*proto.EntryId, error) { //nolint:revive
 	timer := s.newTermQuorumLatency.Timer()
 
-	fencingQuorum := mergeLists(s.shardMetadata.Ensemble, s.shardMetadata.RemovedNodes)
-	fencingQuorumSize := len(fencingQuorum)
-	majority := fencingQuorumSize/2 + 1
+	ensembleQuorum := s.shardMetadata.Ensemble
```

Comment:
The fencing quorum here should also include the removed nodes. We need to make sure there is not another coordinator instance that will fence them in a different term with a different majority.

Reply:
>The fencing quorum here should also include the removed nodes. We need to make sure there is not another coordinator instance that will fence them in a different term with a different majority.

1. Do you mean we might have another coordinator deployment?  
2. Yes, we can keep the logic here and just filter all the unexpected followers. but I am considering if it's breaking the expected quorum. 
for example: 
we have server [sv1,sv2,sv3] and deleted node [sv4].  the swap operation is from `sv4` to `sv3`.  and right now. The `sv3` and `sv4` crashed at the same time.  

then the quorum should be `4/2 +1 = 3` success response. but two crashed servers will let the election constantly fail. 

However, if we keep the quorum equal to `3/2 + 1 =2` success response (don't count removed node). even `sv4` and `sv3` crashed at same time. the leader election can still succeed.